src/components/HomeScreen/HomeScreen.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+
```

Comment:
useless line break between the imports. please remove it

Reply:
fixed